setup.py

```diff
@@ -3,6 +3,13 @@
 
 setup(name='youclid',
       version='0.1',
+      python_requires='>=3.6',
```

Comment:
We use things that require python3.6, so we may as well ensure that the user has it when trying to install